docs/how-to/web-app-rocks/set-up-web-app-rock.rst

```diff
@@ -179,3 +179,173 @@ following snippet to the project file:
 
       For the ``go-framework`` extension, a deb could be needed for example to use an external command in the migration process.
 
+
+Override commands
+-----------------
+
+The ``services`` key follows the :external+pebble:ref:`Pebble layer specification
+<layer-specification>` and defines the entrypoint to your app.
+You can override the default service commands in ``rockcraft.yaml``.
+
+To override a service's command, check the default service
+entrypoint generated by the extension by running ``rockcraft expand-extensions``.
+Then, adapt the command to your needs and declare it in your project file. For
+example:
+
+.. tabs::
+
+   .. group-tab:: Flask
+
+      .. code-block:: yaml
+         :caption: Output of ``rockcraft expand-extensions``
+
+         # ...
+
+         services:
+           flask:
+             override: replace
+             command: /bin/python3 -m gunicorn -c /flask/gunicorn.conf.py app:app -k [ sync ]
+             startup: enabled
+             after:
+               - statsd-exporter
+             user: _daemon_
+
+      To limit the maximum number of pending connections in Gunicorn to 1024, add the
+      following lines to your project file.
+
+      .. code-block:: yaml
+         :caption: rockcraft.yaml
+
+         services:
+           flask:
+             command: /bin/python3 -m gunicorn -c /flask/gunicorn.conf.py app:app --backlog 1024
+               -k [ sync ]
```

Comment:
is this `-k [ sync ]` being on a new line on purpose? If so, on my 1080 screen it might look better if you break starting on `--backlog` instead

<img width="872" height="272" alt="Image" src="https://github.com/user-attachments/assets/483b8ba2-f725-4f3d-9669-1a549df01dc8" />